source/_integrations/generic.markdown

```diff
@@ -15,7 +15,12 @@ Home Assistant will serve the images via its server, making it possible to view
 
 ## Configuration
 
-To enable this camera in your installation, add the following to your `configuration.yaml` file:
+Home Assistant offers the Generic IP Camera through **Configuration -> Integrations -> Generic IP Camera**.
+
+You must enter a URL in at least one of the fields **Still Image URL** or **Stream Source URL**, the others are optional. Click next to continue.
+
```

Comment:
We should explain how the templating of URLs work with the config flow

Reply:
Ok, I've added something which should do the job.